packages/backend/jest.config.ts

```diff
@@ -3,7 +3,7 @@ import type { Config } from 'jest'
 const config: Config = {
   preset: 'ts-jest',
   testEnvironment: '<rootDir>/src/prisma/prismaJestEnvironment.mjs',
-  collectCoverageFrom: ['src/**/*.{ts,tsx}', '!**/seed.ts'],
+  collectCoverageFrom: ['src/**/*.{ts,tsx}'],
```

Comment:
what was it doing?

Reply:
It was an old implementation that was removed long ago, but its instances were forgotten.